recipes-extended/lsb/lsbinitscripts_10.04.bb

```diff
@@ -10,8 +10,12 @@ RCONFLICTS_${PN} = "initscripts-functions"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
 S = "${WORKDIR}/git"
-SRC_URI = "git://github.com/fedora-sysv/initscripts"
+SRC_URI = "git://github.com/fedora-sysv/initscripts \
+           file://0001-lsbinitscripts-replace-quiet-with-q.patch"
```

Comment:
Quick nit: could use a multiline assignment like:
```bash
SRC_URI = "\
    git://... \
    file://... \
"
```

It helps to keep the commit diffs clean. :)

Reply:
Done